crystalball/wsclean.py

```diff
@@ -114,6 +175,44 @@ def import_from_wsclean(wsclean_comp_list, include_regions=[],
                          in wsclean_comps.items()}
         log.info("Selecting up %d brightest sources", num)
 
+    if clean_mask_file:
+        header, clean_mask = load_clean_mask(clean_mask_file)
+        wcs = WCS(header)
+
+        ra = np.rad2deg(wsclean_comps["Ra"]) * u.rad
+        dec = np.rad2deg(wsclean_comps["Dec"]) * u.rad
+        flux = wsclean_comps["I"]
+
+        freq = SpectralQuantity(wsclean_comps["ReferenceFrequency"] * u.Hz,
+                                doppler_rest=wcs.wcs.restfrq * u.Hz,
+                                doppler_convention="radio")
```

Comment:
It would be nice to find some better way to do this conversion as a Radio doppler convention os assumed here.  The restfreq is on the wcs attribute for example. I intuit that the `wcs.spectral` atttribute  (which is a wcs object for the Spectral coordinate axis) should be used, but a simple `wcs.spectral.world_to_pixel`  didn't work for me..